content/help/ops/traffic-management/deploy-guidelines/index.md

```diff
@@ -6,6 +6,113 @@ weight: 5
 
 This section provides specific deployment or configuration guidelines to avoid networking or traffic management issues.
 
+## Virtual service and destination rule host configuration merging
+
+In situations where it is inconvenient to define the complete set of route rules or policies for a particular
+host in a single `VirtualService` or `DestinationRule` resource, it may be possible to incrementally specify
+the configuration for the host in separate resources, which will be merged when they are applied.
```

Comment:
Long sentence. Suggest breaking. Also suggest adding note about this being experimental and that user should read the caveats section before trying.

Reply:
Also qualify that virtual service merging works only for gateways